core-service/src/main/java/com/crmfoodestablishment/coreservice/dto/MenuDto.java

```diff
@@ -0,0 +1,27 @@
+package com.crmfoodestablishment.coreservice.dto;
+
+import com.crmfoodestablishment.coreservice.entity.Dish;
+import com.crmfoodestablishment.coreservice.entity.Season;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import lombok.Data;
+
+import java.util.List;
+
+@Data
+public class MenuDto {
+
+    private Integer id;
```

Comment:
it is redundant to have id in menuDto
just recommendation: it is possible to use menuRequestBody/menuResponseBody to see difference between dtos

Reply:
I delete variable id in menuDtoLayer.